### DISCUSSION TITLE
Vocês se candidatam em vagas de empresas patrocinadoras de eventos de tecnologia?

### DISCUSSION THREAD

**Discussion by woliveiras** [frontendbr/forum#1599] [Discussão]
Quando vocês vão procurar um emprego novo, costumam olhar páginas de apoio dos eventos e se cadastrar no banco de currículo/nas vagas das empresas que patrocinam eles?

(também tem mais conversa [aqui](https://twitter.com/w_oliveiras/status/1192185244075646978))

**Comment by caioincau** [comment]
Costumo sim, isso pra mim é um bom diferencial pra mim que gosto de participar de eventos.

**Comment by jorgehclinhares** [comment]
Já me candidatei, empresas que se interessam pela comunidade já são diferenciadas.

**Comment by felipefialho** [comment]
Pra mim é um diferencial competitivo também. Mostra inclusive que estão aptas a entender e aplicar o conceito de open source no dia a dia.

**Comment by barraponto** [comment]
Já me candidatei mas não durante o evento.

**Comment by ninetails** [comment]
Pessoalmente levo em consideração muito mais se os devs da empresa vão a eventos/meetups como palestrantes ou participantes que o patrocínio em si.
